Controller/src/LSST/M1M3/SS/Accelerometer/Accelerometer.h

```diff
@@ -15,35 +15,25 @@ namespace LSST {
 namespace M1M3 {
 namespace SS {
 
-class AccelerometerSettings;
 struct SupportFPGAData;
+class AccelerometerSettings;
 class M1M3SSPublisher;
 
 /*!
  * The class used to process accelerometer data.
  */
 class Accelerometer {
-   private:
-    AccelerometerSettings* accelerometerSettings;
-    SupportFPGAData* fpgaData;
-    M1M3SSPublisher* publisher;
-
-    MTM1M3_accelerometerDataC* accelerometerData;
-    MTM1M3_logevent_accelerometerWarningC* accelerometerWarning;
-
    public:
-    /*!
-	 * Instantiates the accelerometer.
-	 * @param[in] accelerometerSettings The accelerometer settings.
-	 * @param[in] fpgaData The fpga data.
-	 * @param[in] publisher The publisher.
-	 */
     Accelerometer(AccelerometerSettings* accelerometerSettings, SupportFPGAData* fpgaData, M1M3SSPublisher* publisher);
-
-    /*!
-	 * Processes currently available accelerometer data and publish it.
-	 */
+    ~Accelerometer() = default;
```

Comment:
Given the use of instance variables that are bare pointers, is the default implementation safe?

I'm guessing there was nothing before, which is no better than what you are adding. But I think it is still worth fixing, if not here, then on a new ticket. (But see my comment about the bare pointers below).

Reply:
Your point is correct. Actually I didn't want to include this default destructors in this PR. Also, several classes do not have destructors and all rely on many pointers. I agree with you and I will create a special ticket to treat this issue. On the mean time, I will remove the default destructor added here.